.github/workflows/workflows.json

```diff
@@ -48,20 +51,17 @@
   "healthcare/fhir",
   "healthcare/hl7v2",
   "kms",
-  "game-servers/snippets",
```

Comment:
Praise: Thank you for alphabetizing this config file.
Suggestion: Changes like this are our of scope of the PR, consider putting reorganization work in a separate PR, especially in this context where individual branch commits will be retained as part of the repository history.

Reply:
Okay, fair enough. I've reverted the alphabetization.